pkg/asset/installconfig/installconfig.go

```diff
@@ -152,7 +152,11 @@ func (a *InstallConfig) finish(ctx context.Context, filename string) error {
 		a.IBMCloud = icibmcloud.NewMetadata(a.Config)
 	}
 	if a.Config.PowerVS != nil {
-		a.PowerVS = icpowervs.NewMetadata(a.Config)
+		metadata, err := icpowervs.NewMetadata(a.Config)
+		if err != nil {
+			return err
+		}
+		a.PowerVS = metadata
```

Comment:
So this is done at the end of the `install-config` asset generation. That means anywhere this asset is used, this should be already initialized.